raiden/network/pathfinding.py

```diff
@@ -395,3 +415,6 @@ def query_paths(
                 else:
                     raise
             log.info(f'PFS rejected our IOU, reason: {error}. Attempting again.')
+
+    # If we got no results after MAX_PATHS_QUERY_ATTEMPTS return empty list of paths
+    return list()
```

Comment:
Hey @jomuel / @palango / @karlb  can you have a look at this? The query_paths function had a bug and if at the end of the iteration of calls to the PFS no response was returned then it had no return value, so it returned `None`. Here I make it return an empty list of paths instead. It seems okay to me but since I don't know the logic of PFS querying I want to make sure it's fine. There is also the possibility of raising an exception there.

Reply:
I think this change should be fine since returning an empty list will fail the transfer AFAIK

Reply:
@jomuel You worked on this. Can you confirm what @rakanalh wrote above?